modules/mobile-tests/helpers/capabilities.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-undef */
+
+const IOS = {
+    // eslint-disable-next-line max-len
+    'appium:app': 'Users/cchitu/Library/Developer/Xcode/DerivedData/jitsi-meet-gpsplsfarleyxebhmcufemivqqbl/Build/Products/Debug-iphonesimulator/jitsi-meet.app',
+    'appium:appiumVersion': '1.22.3',
+    'appium:automationName': 'XCUITest',
+    'appium:newCommandTimeout': 240,
+    platformName: 'iOS',
+    'appium:platformVersion': process.env.PLATFORM_VERSION,
+    'appium:udid': process.env.DEVICE_ID,
+    'appium:xcodeOrgId': 'AJT772J42H',
+    'appium:xcodeSigningId': 'iPhone Developer'
+};
+
+const DEVICE_IOS = {
+    ...IOS,
+    // eslint-disable-next-line max-len
+    'appium:app': '/Users/cchitu/Library/Developer/Xcode/Archives/2022-09-01/JitsiMeet.xcarchive/Products/Applications/jitsi-meet.app'
```

Comment:
This path needs to be predictable and not tied to a user.

Reply:
Can we use this? 
https://stackoverflow.com/questions/44418912/xcodebuild-archive-path-of-newly-created-archive

Reply:
Yeah, that would do it.